tests/test_ec2_metadata.py

```diff
@@ -16,11 +17,20 @@ def clear_it():
 
 
 example_mac = "00:11:22:33:44:55"
+example_token = "AABBCC"
 
 
 # EC2Metadata tests
 
 
+def _mock_token_request(requests_mock):
```

Comment:
There's a better way to push this into a new pytest fixture rather than calling it within every single function below. If you can't figure it out, I'll push to the branch when I come back to review.